datalad/resources/website/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>Datalad Repository</title>
+        <link rel="shortcut icon" type="image/x-icon" href=".git/datalad/web/assets/images/favicon-16x16.png" />
+        <link rel="stylesheet" type="text/css" href=".git/datalad/web/assets/css/jquery.dataTables.css">
+        <style type="text/css" class="init">
```

Comment:
Overwriting (vs merely overriding) CSS in an `<style>` tag and acquiring that element via a class name would be some pretty messed-up-foo IMO.

If I were to guess, this is either an accidental copy-paste or a leftover from a clever/insane approach to providing an interactive example.

In either case, I'd recommend removing it, as I was surprised it's even valid (and I think it only is by accident, by way of `class` being a global attribute. The specs (4, 5, XHTML, nor the Living Standard) make no mention of any semantic difference or purpose to using `class` on a `<style>` tag).

If everything breaks, obviously keep it, and please let me know, as I will be /very/ interested.

---Alex